global.json

```diff
@@ -7,10 +7,10 @@
     "runtimes": {
       "dotnet/x64": [
         "2.1.18",
-        "$(MicrosoftNETCoreAppInternalVersion)"
+        "$(MicrosoftNETCoreBrowserDebugHostTransportVersion)"
```

Comment:
Nit, this package seems super arbitrary. Would it make more sense to use something like `MicrosoftNETCoreAppRuntimewinx64Version`

Reply:
No, that wouldn't make sense. Need a package we depend on anyway that doesn't ship. The options are this one or Microsoft.Extensions.HostFactoryResolver.Sources and @jkoritzinsky picked this one. Fine by me -- there's no difference between them as far as their possibility of shipping at some future date (neither will).